gptel.el

```diff
@@ -968,7 +970,8 @@ Model parameters can be let-bound around calls to this function."
   (declare (indent 1))
   (let* ((gptel--system-message
           ;Add context chunks to system message if required
-          (if (and gptel-context--alist
+          (if (and system
```

Comment:
If there's no system message but there are context chunks (and `gptel-use-context` is set to `system`), they should still be supplied as the context.